fachhochschule-vorarlberg-author-date.csl

```diff
@@ -151,60 +140,58 @@
         </choose>
       </else-if>
       <else>
-        <text variable="title" font-style="normal" prefix=" " suffix=" "/>
+        <text variable="title" prefix=" " suffix=". "/>
       </else>
     </choose>
   </macro>
   <macro name="collection-title">
     <text variable="collection-title" prefix=" (= " suffix=")"/>
     <choose>
-      <if variable="page URL volume" match="none">
+      <if variable="page " match="none">
         <text value=". "/>
       </if>
     </choose>
   </macro>
   <macro name="access">
     <choose>
-      <if type="webpage post-weblog" match="any">
-        <choose>
-          <if variable="URL">
-            <group delimiter=" " suffix=":">
-              <text term="online" text-case="capitalize-first"/>
-              <text value="im"/>
-              <text term="internet"/>
-            </group>
-            <text variable="URL"/>
-            <group prefix=" (" delimiter=" " suffix=").">
-              <text term="accessed" suffix=":"/>
-              <date variable="accessed">
-                <date-part name="day" form="numeric-leading-zeros" suffix="."/>
-                <date-part name="month" suffix="." form="numeric-leading-zeros"/>
-                <date-part name="year" form="long"/>
-              </date>
-            </group>
-          </if>
-        </choose>
+      <if variable="DOI">
+        <group suffix=": DOI: ">
+          <text term="available at"/>
+        </group>
+        <text variable="DOI"/>
       </if>
+      <else-if variable="URL">
+        <group suffix=": URL: ">
+          <text term="available at"/>
+        </group>
+        <text variable="URL"/>
+      </else-if>
```

Comment:
@cookie365, groups with just a single child element are redundant. It's better to e.g. change 

```xml
      <if variable="DOI">
        <group suffix=": DOI: ">
          <text term="available at"/>
        </group>
        <text variable="DOI"/>
      </if>
```

to

```xml
      <if variable="DOI">
        <group delimiter=": ">
          <text term="available at"/>
          <text value="DOI"/>
          <text variable="DOI"/>
        </group>
      </if>
```